include/tvm/runtime/sgx/trusted.h

```diff
@@ -0,0 +1,18 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file trusted.h
+ * \brief TVM SGX trusted API.
+ */
+#ifndef TVM_RUNTIME_SGX_TRUSTED_H_
+#define TVM_RUNTIME_SGX_TRUSTED_H_
+
+#include <sgx_edger8r.h>
+
+extern "C" {
+sgx_status_t ocall_tvm_thread_group_launch(int num_workers, void* cb);
```

Comment:
if these functions are not user facing(but used by internal), we could just keep them in src/runtime/sgx, instead of include